build/HarfBuzzSharp.props

```diff
@@ -1,7 +1,12 @@
 ﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <PackageReference Include="HarfBuzzSharp" Version="2.8.2.3" />
-    <PackageReference Condition="'$(IncludeLinuxSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.Linux" Version="2.8.2.3" />
-    <PackageReference Condition="'$(IncludeWasmSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.WebAssembly" Version="2.8.2.3" />
+    <PackageReference Include="HarfBuzzSharp" Version="7.3.0" />
+    <PackageReference Condition="'$(IncludeWindowsSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.Win32" Version="7.3.0" />
+    <PackageReference Condition="'$(IncludeLinuxSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.Linux" Version="7.3.0" />
+    <PackageReference Condition="'$(IncludeMacSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.macOS" Version="7.3.0" />
+    <PackageReference Condition="'$(IncludeWasmSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.WebAssembly" Version="7.3.0" />
+    <PackageReference Condition="'$(IncludeAndroidSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.Android" Version="7.3.0" />
+    <PackageReference Condition="'$(IncludeIOSSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.iOS" Version="7.3.0" />
+    <PackageReference Condition="'$(IncludeTizenSkia)' == 'true'" Include="HarfBuzzSharp.NativeAssets.Tizen" Version="7.3.0" />
```

Comment:
Everything but WASM and Linux should still be included automatically. You can see from dependencies page here: https://www.nuget.org/packages/HarfBuzzSharp

.NETStandard 2.0 has Win32 and macOS
And .NET 6 has mobile platforms as well.

Reply:
Hmm so it looks like everything is referenced automatically

Reply:
https://github.com/mono/SkiaSharp/blob/main/binding/IncludeNativeAssets.HarfBuzzSharp.targets

Reply:
Yes, everything automatically, except linux and non-blazor browser. But we already cover these platforms manually.